src/server/server_family.cc

```diff
@@ -2423,6 +2423,8 @@ string ServerFamily::FormatInfoMetrics(const Metrics& m, std::string_view sectio
       }
     }
     append("table_used_memory", total.table_mem_usage);
+    append("total_system_memory", total_host_memory);
```

Comment:
let's take it offline - it's news to me that sidekiq requires it - I want to understand more.
exposing free host memory is tricky - it depends on many environmental factors so I  prefer to avoid it if possible

Reply:
Sure.

It's not actually sidekiq that needs it, there is a method in a mastodon module that uses the stats to determine whether sidekiq is usable, https://github.com/mastodon/mastodon/blob/main/app/workers/scheduler/self_destruct_scheduler.rb#L22-L26